samples/image/image.cpp

```diff
@@ -1161,11 +1161,13 @@ MyFrame::MyFrame()
     SetStatusWidths( 2, widths );
 #endif // wxUSE_STATUSBAR
 
-    m_canvas = new MyCanvas( this, wxID_ANY, wxPoint(0,0), wxSize(10,10) );
-
-    // 500 width * 2750 height
-    m_canvas->SetScrollbars( 10, 10, 50, 275 );
+    m_canvas = new MyCanvas(this);
+    wxSize cz = m_canvas->GetDrawingSize();
+    m_canvas->SetScrollbars(10, 10, cz.GetWidth() / 10, cz.GetHeight() / 10);
```

Comment:
No need for `FromDIP()` here?

Reply:
No, the `SetScrollbars` arguments (`pixelsPerUnit` * `noUnits`) should match the number of pixels of the canvas.

It is actually not per-monitor DPI aware, so I'll add a DPI event handler and update `SetScrollbars` when the canvas size changes.